packages/next/src/views/Root/index.tsx

```diff
@@ -66,24 +66,29 @@ export const RootPage = async ({
 
   let dbHasUser = false
 
+  const initPageResult = await initPage(initPageOptions)
+
+  dbHasUser = await initPageResult?.req.payload.db
```

Comment:
i do not know that i like this here - this is akin to the `init` operation, right? isn't that being run somewhere else already?